src/libponyrt/actor/actor.c

```diff
@@ -15,6 +15,10 @@
 #include <valgrind/helgrind.h>
 #endif
 
+#define INITIAL_BATCH 100
+#define INCR_BATCH 50
+#define DECR_BATCH 25
```

Comment:
INCR_BATCH and DECR_BATCH could be exposed as tuneable command line options but that might be too low level to expose. I think keeping private for now where "in the know" folks could use to try and tune makes sense. These values were choosen mostly "at random" and have worked out for us thus far.